ci/src/artifacts/upload.bzl

```diff
@@ -23,9 +26,19 @@ def _upload_artifact_impl(ctx):
         VERSION_FILE={version_file}
         export RCLONE={rclone}
         export VERSION=$(cat $VERSION_FILE)
+
+        # upload any artifacts specified on the command line
+        while [[ $# -gt 0 ]]
+        do
+            p=$(cd "$BUILD_WORKSPACE_DIRECTORY"; realpath "$1")
+            shift
+            {uploader} "$p"
```

Comment:
probably worth setting the uploader once as e.g `UPLOADER={upload}` and then reuse it throughout instead of interpolating twice